tests/baselines/reference/moduleAugmentationCollidingNamesInAugmentation1.js

```diff
@@ -0,0 +1,75 @@
+//// [tests/cases/compiler/moduleAugmentationCollidingNamesInAugmentation1.ts] ////
+
+//// [map1.ts]
+
+import { Observable } from "./observable"
+
+(<any>Observable.prototype).map = function() { }
```

Comment:
Do you need to cast to `any` here? I would think that the type of `prototype` would be augmented by the interface correctly.